setup.py

```diff
@@ -1,103 +1,76 @@
-
-
-def __package_files(directory):
-    """
-    Collect the package files.
-    """
-    paths = []
-    for (path, _, filenames) in os.walk(directory):
-        for filename in filenames:
-            paths.append(os.path.join('..', path, filename))
-    return paths
-
-
-def __package_data():
-    """
-    Return a list of package data.
-    """
-    __compile_po_files()
-    data = ['glade/*.glade', 'resource/*']
-    data.extend(__package_files('safeeyes/config'))
-    data.extend(__package_files('safeeyes/plugins'))
-    data.extend(__package_files('safeeyes/platform'))
-    return data
-
-setuptools.setup(
-    name="safeeyes",
-    version="2.2.2",
-    description="Protect your eyes from eye strain using this continuous breaks reminder.",
-    long_description=long_description,
-    long_description_content_type="text/markdown",
-    author="Gobinath Loganathan",
-    author_email="slgobinath@gmail.com",
-    url="https://github.com/slgobinath/SafeEyes",
-    download_url="https://github.com/slgobinath/SafeEyes/archive/v2.2.2.tar.gz",
-    packages=setuptools.find_packages(),
-    package_data={'safeeyes': __package_data()},
-    data_files=__data_files(),
-    install_requires=requires,
-    entry_points={'console_scripts': ['safeeyes = safeeyes.__main__:main']},
-    keywords='linux utility health eye-strain safe-eyes',
-    classifiers=[
-        "Operating System :: POSIX :: Linux",
-        "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
-        "Development Status :: 5 - Production/Stable",
-        "Environment :: X11 Applications :: GTK",
-        "Intended Audience :: End Users/Desktop",
-        "Topic :: Utilities"] + [('Programming Language :: Python :: %s' % x) for x in '3 3.5 3.6 3.7 3.8 3.9'.split()]
+#!/usr/bin/python3
+
+import os
+
+from pathlib import Path
+from setuptools import Command, setup
+from setuptools.command.build import build as orig_build
+
+class build(orig_build):
+    sub_commands = [('build_mo', None), *orig_build.sub_commands]
+
+
+class build_mo(Command):
+    description = 'Compile .po files into .mo files'
+
+    files = None
+
+    def initialize_options(self):
+        self.files = None
+        self.editable_mode = False
+        self.build_lib = None
+
+    def finalize_options(self):
+        self.set_undefined_options("build_py", ("build_lib", "build_lib"))
+        pass
+
+    def run(self):
+        files = self._get_files()
+
+        for build_file, source_file in files.items():
+            if not self.editable_mode:
+                # Parent directory required for msgfmt to work correctly
+                Path(build_file).parent.mkdir(parents=True, exist_ok=True)
+            self.spawn(['msgfmt', source_file, '-o', build_file])
+
+    def _get_files(self):
+        if self.files is not None:
+            return self.files
+
+        files = {}
+
+        localedir = 'safeeyes/config/locale'
```

Comment:
(Minor) Look at `pathlib` package, it is convenient `os` replacement for some use cases